mk/golang.mk

```diff
@@ -68,6 +68,10 @@ test_go_fmt:
 .PHONY: test_go_fmt
 TEST_GO += test_go_fmt
 
+test_go_mod: 
+	go mod tidy
```

Comment:
`test_go_mod` should check after running `go mod tidy`, whether `git diff` has any resulted changes, similar to what `test-go-fmt` does
https://github.com/TRON-US/go-btfs/blob/master/bin/test-go-fmt

This way devs wouldn't commit out-of-date deps

Reply:
ah ok i understand.  thanks.  i will redo the PR.